README.md

```diff
@@ -143,3 +143,24 @@ func main() {
 
 The main examples have been moved - you can find them in their [own repository](https://github.com/fyne-io/examples/).
 
+## Non-standard library dependencies
+
+Fyne requires the following imports which are not included in the go standard library:
+
+* github.com/go-gl/gl/v3.2-core/gl
+* github.com/go-gl/glfw/v3.2/glfw
+* github.com/golang/freetype
+* github.com/srwiley/oksvg
+* github.com/srwiley/rasterx
+* golang.org/x/image/font
+
+These can be included in your gopath by the following 'get' commands:
+
+* "go get github.com/go-gl/gl/v3.2-core/gl"
+* "go get github.com/go-gl/glfw/v3.2/glfw"
+* "go get github.com/golang/freetype"
+* "go get github.com/srwiley/oksvg"
+* "go get github.com/srwiley/rasterx"
+* "go get golang.org/x/image/font"
+
+You will also require a GCC compiler if you do not have one, this can be included by installing TDM-GCC.
```

Comment:
This assumes a Windows user, which is not necessarily the case. This could very well be a Darwin/Linux/*BSD/other user.

Also, TDM-GCC is not necessarily the best way to handle Windows CGO support (I had not even heard of it before now). If we even think that we need to explain this, we should link to the Go wiki: https://github.com/golang/go/wiki/WindowsBuild

Reply:
I wonder if this could be phrased more along the lines of "Fyne uses CGo, which requires a C compiler to be installed. If you don't already have one installed you can ...".
After all it's not like we require a C compiler directly so it would be good if this sounds like less of a special case.

Reply:
Also, we may be able to remove CGo dependency on windows in the near future :smile:

https://github.com/go-gl/glow/pull/102